operator/pkg/controlplane/etcd/etcd.go

```diff
@@ -25,19 +25,31 @@ func EnsureKarmadaEtcd(client clientset.Interface, cfg *operatorv1alpha1.LocalEt
 }
 
 func installKarmadaEtcd(client clientset.Interface, name, namespace string, cfg *operatorv1alpha1.LocalEtcd) error {
+	// if the number of etcd is greater than one, we need to concatenate the
+	// peerURL for each member cluster.
+	// memberName is podName generated by etcd statefuleset: ${statefulsetName}-index
+	// memberPeerURL uses the etcd peer headless service name: ${podName}.${namespace}.svc.cluster.local:2380
+	initialClusters := make([]string, *cfg.Replicas)
+	for index := range initialClusters {
+		memberName := fmt.Sprintf("%s-%d", util.KarmadaEtcdName(name), index)
+		memberPeerURL := fmt.Sprintf("%s.%s.svc.cluster.local:%v", memberName, namespace, constants.EtcdListenPeerPort)
```

Comment:
There seems an error when starting etcd, could you please take a look? @calvin0327 

```
{"level":"warn","ts":"2023-04-10T06:20:24.484Z","caller":"etcdmain/etcd.go:146","msg":"failed to start etcd","error":"error setting up initial cluster: URL scheme must be http, https, unix, or unixs: karmada-demo-etcd-0.test.svc.cluster.local:2380"}
{"level":"fatal","ts":"2023-04-10T06:20:24.484Z","caller":"etcdmain/etcd.go:204","msg":"discovery failed","error":"error setting up initial cluster: URL scheme must be http, https, unix, or unixs: karmada-demo-etcd-0.test.svc.cluster.local:2380","stacktrace":"go.etcd.io/etcd/server/v3/etcdmain.startEtcdOrProxyV2\n\t/go/src/go.etcd.io/etcd/release/etcd/server/etcdmain/etcd.go:204\ngo.etcd.io/etcd/server/v3/etcdmain.Main\n\t/go/src/go.etcd.io/etcd/release/etcd/server/etcdmain/main.go:40\nmain.main\n\t/go/src/go.etcd.io/etcd/release/etcd/server/main.go:32\nruntime.main\n\t/go/gos/go1.16.15/src/runtime/proc.go:225"}
```